src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,17 @@
-export const TodoList = () => {};
+import React from "react";
+import { Todo } from "../../Types/Todo";
+import { TodoInfo } from "../TodoInfo";
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map((todo) => (
+        <TodoInfo todo={todo} />
```

Comment:
Each child in a list should have a unique `key` prop to help React identify which items have changed. Add a `key` prop to the `TodoInfo` component, using a unique value like `todo.id`.